.github/workflows/smoke-test-ipv6.yaml

```diff
@@ -73,16 +73,14 @@ jobs:
           kubectl apply -f ${{ env.CONFORMANCE_TEMPLATE }}
           kubectl wait --for=condition=Available --all deployment --timeout=${{ env.TIMEOUT }}
 
-      - name: Dump cilium related logs and events
+      - name: Capture cilium sysdump
         if: ${{ failure() }}
         run: |
-          kubectl -n kube-system describe daemonsets.apps cilium
-          kubectl -n kube-system logs daemonset/cilium --all-containers --since=${{ env.LOG_TIME }}
-          kubectl -n kube-system logs service/kube-dns --all-containers --since=${{ env.LOG_TIME }}
+          curl -sLO https://github.com/cilium/cilium-sysdump/releases/latest/download/cilium-sysdump.zip
+          python cilium-sysdump.zip --output cilium-sysdump-out
 
-      - name: Dump connectivity related logs and events
+      - uses: actions/upload-artifact@v2
```

Comment:
Can we make this part of `Capture cilium sysdump` by removing the dash?

Reply:
I am not sure if I understand your comment completely. Do you mean to merge this upload into previous step as well ? I don't think it's possible as [actions/upload-artifact(https://github.com/actions/upload-artifact) is kind of independent building block in GA.

Reply:
Ah, I see. We may want to name that step then. I'm fine as is though.